platform/macos/CHANGELOG.md

```diff
@@ -40,6 +34,7 @@
 * Fixed a crash when switching between two styles having layers with the same identifier but different layer types. ([#12432](https://github.com/mapbox/mapbox-gl-native/issues/12432))
 * Added a new option to `MGLSymbolPlacement`, `MGLSymbolPlacementLineCenter`, that places the label relative to the center of the geometry. ([#12337](https://github.com/mapbox/mapbox-gl-native/pull/12337))
 * Added a `MGLShapeSourceOptionLineDistanceMetrics` property that enables or disables calculating line distance metrics.
+* Add `symbol-z-order` symbol layout property to style specification. [#12783](https://github.com/mapbox/mapbox-gl-native/pull/12783)
```

Comment:
This change did not make it into the release-frappé branch, as far as I can tell.